gateway_guide.html

```diff
@@ -409,267 +403,295 @@ <h2 id="generating-source-and-destination-tags">Generating Source and Destinatio
-  }
-}
-</code></pre>
-<p><em>(<strong>Reminder:</strong> Don't send your secret to a server you do not control.)</em></p>
-<p>Response:</p>
-<pre><code>201 Created
+<pre><code>POST http://localhost:8088/
 {
-  "success": true,
-  "trustline": {
-    "account": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
-    "limit": "0",
-    "currency": "USD",
-    "counterparty": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
-    "account_allows_rippling": true,
-    "account_trustline_frozen": false,
-    "authorized": true
-  },
-  "hash": "4509288EE17F01C83FC7D45850EB066A795EE5DBA17BB4DC98DD4023D31EEE5B",
-  "ledger": "11158585",
-  "state": "validated"
+    "method": "submit",
+    "params": [
+        {
+            "secret": "sn3nxiW7v8KXzPzAqzyHXbSSKNuN9",
+            "tx_json": {
+                "Account": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+                "Fee": "15000",
+                "TransactionType": "TrustSet",
+                "LimitAmount": {
+                    "currency": "USD",
+                    "issuer": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+                    "value": 0
+                },
+                "Flags": 65536
+            }
+        }
+    ]
 }
 </code></pre>
-<p>A successful response shows <code>"state": "validated"</code> when the change has been accepted into a validated Ripple ledger.</p>
+<p><em>(<strong>Reminder:</strong> Don't send your secret to a server you do not control.)</em> </p>
 <h2 id="robustly-monitoring-for-payments">Robustly Monitoring for Payments</h2>
 <p>In order to robustly monitor incoming payments, gateways should do the following:</p>
 <ul>
 <li>Keep a record of the most-recently-processed transaction and ledger. That way, if you temporarily lose connectivity, you know how far to go back.</li>
 <li>Check the result code of every incoming payment. Some payments go into the ledger to charge an anti-spam fee, even though they failed. Only transactions with the result code <code>tesSUCCESS</code> can change non-XRP balances. Only transactions from a validated ledger are final.</li>
-<li><a href="https://ripple.com/files/GB-2014-06.pdf" title="Partial Payment Flag Gateway Bulletin">Look out for Partial Payments</a>. If an incoming transaction has a <code>destination_balance_changes</code> field (Ripple-REST) or a <code>meta.delivered_amount</code> field (WebSocket/JSON-RPC), then use that to see how much money <em>actually</em> got delivered to the destination account. Payments with the partial-payment flag enabled are considered "successful" if any non-zero amount is delivered, even miniscule amounts. (The flag is called <code>"partial_payment": true</code> in REST, and <code>tfPartialPayment</code> in WebSocket/JSON-RPC) </li>
+<li><a href="https://ripple.com/files/GB-2014-06.pdf" title="Partial Payment Flag Gateway Bulletin">Look out for Partial Payments</a>. Payments with the partial-payment flag enabled can be considered "successful" if any non-zero amount is delivered, even miniscule amounts.<ul>
```

Comment:
KC link for the gateway bulletin: https://ripple.com/knowledge_center/partial-payment-flag/